pkg/operator/ceph/nfs/config.go

```diff
@@ -47,18 +48,22 @@ func getNFSNodeID(n *cephv1.CephNFS, name string) string {
 	return fmt.Sprintf("%s.%s", n.Name, name)
 }
 
-func getGaneshaConfigObject(clusterName string) string {
-	return fmt.Sprintf("conf-nfs.%s", clusterName)
+func getGaneshaConfigObject(n *cephv1.CephNFS, version cephver.CephVersion, name string) string {
+	if version.IsAtLeastOctopus() {
+		return fmt.Sprintf("conf-nfs.ganesha-%s", n.Name)
+	} else {
```

Comment:
No need for the `else`, use `fmt.Sprintf("conf-%s", getNFSNodeID(n, name))` as the default return.

Reply:
Done